pytroll_collectors/tests/data/segments.ini

```diff
@@ -18,3 +18,16 @@ publish_topic = /pub/foo/bar
 timeliness = 1500
 time_name = start_time
 variable_tags = end_time,end_frac_sec,creation_time
+
+[himawari-8]
+pattern = IMG_DK01{channel_name:3s}_{start_time:%Y%m%d%H%M}_{segment:0>3s}
+critical_files = B01:001-010,B02:001-010,B04:001-010,B05:001-010,B09:001-010,B10:001-010,B11:001-010,B14:001-010,B16:001-010,IR1:001-010,IR2:001-010,IR3:001-010,IR4:001-010,VIS:001-010
+wanted_files = B01:001-010,B02:001-010,B04:001-010,B05:001-010,B06:001-010,B09:001-010,B10:001-010,B11:001-010,B12:001-010,B14:001-010,B16:001-010,IR1:001-010,IR2:001-010,IR3:001-010,IR4:001-010,VIS:001-010
+all_files = B01:001-010,B02:001-010,B04:001-010,B05:001-010,B06:001-010,B09:001-010,B10:001-010,B11:001-010,B12:001-010,B14:001-010,B16:001-010,IR1:001-010,IR2:001-010,IR3:001-010,IR4:001-010,VIS:001-010
+topics = /himawari/H8/L1
+publish_topic = /himawari/H8/segment
+timeliness = 1200
+time_name = start_time
+variable_tags = start_minute
```

Comment:
This isn't needed anymore, right ?

Reply:
Thanks! Removed `variable_tags` from both Himwari configs (example template and tests).